pyiron_contrib/atomistics/atomicrex/base.py

```diff
@@ -259,6 +253,30 @@ def potential_as_pd_df(self):
         return self.potential._potential_as_pd_df(job=self)
 
 
+    #### PotentialFit methods
+    def _add_training_data(self, container: TrainingContainer) -> None:
+        self.structures.add_training_data(container)
+
+    def _get_training_data(self) -> TrainingStorage:
+        return self.structures.get_training_data()
+
+    def _get_predicted_data(self) -> FlattenedStorage:
+        return self.structures.get_predicted_data()
+
+    def get_lammps_potential(self) -> pd.DataFrame:
+        """
+        Return a pyiron compatible dataframe that defines a potential to be used with a Lammps job (or subclass
+        thereof).
+
+        Returns:
+            DataFrame: contains potential information to be used with a Lammps job.
+        """
+        return self.potential_as_pd_df()
+
+    @property
+    def plot(self):
+        return self.structures.plot
```

Comment:
I think until we have settled on a few generic plots for potentials, this should not be overwritten or at least it should be doing the same as original `PotentialFit.plot`.

Reply:
I also changed the PotentialFit plot method, because it felt weird to have a .plot on TrainingStorage and on the PotentialFit job class doing 2 completely different things. One is the interface to lots of plotting functionality which I really like and the one on PotentialFit only plots energies, which is nice but quite limited.

IMO we should change plot in PotentialFit to act as an interface similar to the TrainingStorage